types/react-native-dotenv/tsconfig.json

```diff
@@ -3,7 +3,7 @@
         "module": "commonjs",
         "lib": ["es6"],
         "esModuleInterop": true,
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```

Comment:
Does this need to be changed? I'm not seeing how this change necessitates disabling `noImplicitAny`.

Reply:
You are correct, this doesn't need to be changed :).